cells.gemspec

```diff
@@ -19,8 +19,11 @@ Gem::Specification.new do |spec|
   spec.metadata['source_code_uri'] = spec.homepage
   spec.metadata['wiki_uri'] = "#{spec.homepage}/wiki"
 
-  spec.files         = `git ls-files`.split("\n")
-  spec.test_files    = `git ls-files -- {test}/*`.split("\n")
+  spec.files = Dir.chdir(__dir__) do
+    `git ls-files -z`.split("\x0").reject do |file|
+      file.start_with?(*%w[.git Gemfile Rakefile TODO cells.gemspec test])
```

Comment:
I think we should keep `cells.gemspec`, it helps while debugging.

Reply:
Updated in [`1fc116d`](https://github.com/trailblazer/cells/pull/503/commits/1fc116d41e9af5bd4714ade7714d91947da5a7a9).

Reply:
@ yogeshjain999 what do you think? Should we merge this in?

Reply:
Sorry I missed this, LGTM 🍻

Reply:
Thanks.